### DISCUSSION TITLE
How do I choose the mode/backend?

### DISCUSSION THREAD

**Discussion by oatmealm** [daedsidog/evedel#11] [Q&A] [answered]
Does evedel only uses whatever is defined in `gptel-model` as the default? Can I set it directly or need to define as a buffer local value?

**Comment by daedsidog** [comment, marked as answer]
Right now, Evedel uses only `gptel-request`, so you do it in the same manner you do it in gptel, by binding `gptel-model` and `gptel-backend`.

> Can I set it directly or need to define as a buffer local value?

Yes, same way as you would for other gptel commands.

You can read more about it by consulting the documentation of `gptel-request`.
